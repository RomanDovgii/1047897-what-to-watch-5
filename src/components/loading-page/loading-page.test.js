import React from "react";
import renderer from "react-test-renderer";
import LoadingPage from "./loading-page";

it(
    `Should render correct LoadingPage`,
    () => {
      const tree = renderer
      .create(
          <LoadingPage/>
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
