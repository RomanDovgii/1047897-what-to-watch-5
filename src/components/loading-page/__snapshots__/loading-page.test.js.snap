// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct LoadingPage 1`] = `
<section
  style={
    Object {
      "alignItems": "center",
      "backgroundImage": "linear-gradient(-180deg,#180202 0,#000 100%)",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
      "left": "0",
      "position": "absolute",
      "top": "0",
      "width": "100%",
    }
  }
>
  <p
    style={
      Object {
        "color": "#ffffff",
        "fontSize": "100px",
      }
    }
  >
    Loading...
  </p>
</section>
`;
